src/api/depositV2.js

```diff
@@ -21,7 +23,7 @@ const validateArg0 = validateWithJoi(schema)('INVALID_METHOD_ARGUMENT')({
 const endpoint = '/v1/trading/deposits'
 
 module.exports = async (dvf, data, nonce, signature) => {
-  const { token, amount } = validateArg0(data)
+  const { token, amount, useProxiedContract = false } = validateArg0(data)
```

Comment:
Do we want to put the default value here or in the Joi schema ? (not sure in which case we use which)